icepyx/tests/test_behind_NSIDC_API_login.py

```diff
@@ -2,41 +2,31 @@
 import pytest
 import warnings
 
-# from unittest.mock import patch
-# import mock
-# import builtins
-# import getpass
-import os
 
+# Misc notes and needed tests
 # test avail data and subsetting success for each input type (kml, shp, list of coords, bbox)
-# check that agent key is added in event of no subsetting
-# check that downloaded data is subset
+# check that downloaded data is subset? or is this an NSIDC level test so long as we verify the right info is submitted?
 
 
 @pytest.fixture
-def reg_a(scope="module"):
+def reg(scope="module"):
     return ipx.Query("ATL06", [-55, 68, -48, 71], ["2019-02-22", "2019-02-28"])
 
 
-# @patch('my_module.__get_input', return_value='y')
-
-
+# QUESTION: should we be testing independently to make sure the session starts, or is the fixture sufficient test of that?
```

Comment:
@weiji14

Reply:
I think it's ok to have a fixture that sets up the session, it's similar to the example at https://docs.pytest.org/en/latest/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session where the 'module' scope fixture connects to a server. So no, we don't need an independent test to ensure that the session starts.